### ISSUE TITLE
🐛 [firebase_crashlytics] Warning to upgrade

### ISSUE THREAD

**Issue by mulderpf** [FirebaseExtended/flutterfire#3400]
## Bug report

**Describe the bug**
Using v0.1.4+1 of the plug-in, I now get a warning in my Crashlytics console to upgrade:

> Please upgrade to the generally available Firebase Crashlytics SDK to continue getting crash reports in the Firebase console past November 15, 2020.

I have followed all of the instructions to install Crashlytics and crashes are being reported currently.

### Steps to reproduce

1. Follow the installation instructions for using Crashlytics
2. Cause a crash in the app
3. Navigate to the Crashlytics page in the Firebase console
4. Observe the warning.

### Expected behavior

No warning happens and crashes continue to be reported beyond 15 November.

---

**Comment by matpag** [comment]
The code must be updated to remove Fabric references and switch to the new Firebase Crashlytics https://firebase.google.com/docs/crashlytics/upgrade-sdk

**Comment by darshankawar** [comment]
Hi @mulderpf,
Per @matpag's comment, please follow-up on the WIP PR for updates and any questions.
Thanks.

**Comment by Salakar** [comment]
A new Crashlytics plugin has now landed on master which should solve this issue.

Checkout the migration guide to get started on upgrading to the new Crashlytics plugin: https://firebase.flutter.dev/docs/migration/

**Comment by realappie** [comment]
@Salakar I just added the `firebase_crashlytics` plugin as described in the docs, my `pubspec.yaml` version is set to `^0.2.0-dev.5` yet once I tested my first crash I got a orange banner on top of my firebase crashlytics page with the following message.

> "Please upgrade to the generally available Firebase Crashlytics SDK to continue receiving crash reports in the Firebase console after 15 November 2020. Learn more"

Is this just a general message or is the latest `firebase_crashlytics` version not using the "generally available Firebase Crashlytics SDK" under the hood. Some clarification would be great, thanks.
